src/txn/txn_ckpt.c

```diff
@@ -732,7 +732,8 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
     WT_TXN_GLOBAL *txn_global;
     WT_TXN_ISOLATION saved_isolation;
     wt_timestamp_t ckpt_tmp_ts;
-    uint64_t fsync_duration_usecs, generation, time_start, time_stop;
+    uint64_t fsync_duration_usecs, generation, time_start, time_stop, time_start_hs, time_stop_hs,
```

Comment:
I'd probably rename `time_start` and `time_stop` to something that includes the word `fsync`. They're only named like that since they were the only ones in the function.

Also, declarations on the same line should be in alphabetical order and shouldn't wrap to a newline.

Reply:
okay! 👌 done